src/components/posts/Comment.js

```diff
@@ -3,14 +3,18 @@ import { formatDate } from '../../Helper/dateFormatter';
 export default function Comment({ comment }) {
 	return (
 		<li>
-			<div className='name-time-container'>
-				<div className='name'>
-					{comment.profile.firstName + ' '}
-					{comment.profile.lastName}
+			<div className='comment-container'>
+				<div className='name-time-container'>
+					<div className='name'>
+						{comment.profile.firstName + ' '}
```

Comment:
Did you try to get the space in-between the first name and last name by adding a space in the JSX?

Something like: 
{comment.profile.firstName} {comment.profile.lastName}
```js

```